sys/include/net/gcoap.h

```diff
@@ -8,7 +8,7 @@
  */
 
 /**
- * @defgroup    net_gcoap  CoAP
+ * @defgroup    net_gcoap  Gcoap
```

Comment:
Also, shouldn'd it be `GCoAP` or `gCoAP`?

Reply:
Like nanocoap, the canonical spelling is "gcoap" -- all lower case, for simplicity. The only reason for the captitalization here is to make it easier for the user to find in the documentation groups beneath Networking. The title has been spelled "CoAP" because that is the canonical spelling and the first letter is capitalized. I also used the same transformation in the documentation for nanocoap to "Nanocoap".

Reply:
Ok